### PR TITLE
Add boilerplate code

### PR DESCRIPTION
**Pull request by aalmiramolla**
Nothing to comment

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+SHELL=/bin/bash
+DOCKER_COMPOSE=docker-compose
```
Comment (serfer2) [inline Makefile:2]: Al probarlo, me daba error. Entonces me he acordado: las nuevas versiones de Docker ya llevan compose de serie ([doc](https://docs.docker.com/compose/)). Ya no es necesario usar `docker-compose`. 
El comando es: `docker compose`.
¿Qué te parece si lo dejamos listo para el nuevo?

File: config/django.dev.env [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Django
+DJANGO_SETTINGS_MODULE=meetupselector.settings.dev
+SECRET_KEY=askdfj98asdf0-asdyf7yhaasdfa87asdf723*
```
Comment (sh1l0n) [inline config/django.dev.env:3]: Secrets should not be pushed into repo

File: config/django.prod.env [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Django
+DJANGO_SETTINGS_MODULE=meetupselector.settings.prod
+SECRET_KEY=askdfj98asdf0-asdyf7yha)asdfa87asdf723*
```
Comment (sh1l0n) [inline config/django.prod.env:3]: Secret

File: config/postgresql.env [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# PostgreSQL
+POSTGRES_DB=meetupselector
```
Comment (sh1l0n) [inline config/postgresql.env:2]: Demo account?

File: meetupselector/settings/dev.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import warnings
+
+from .default import *  # noqa
+
+DEBUG = True
+
+SECRET_KEY = "GgL2XYgpsdSpHvDQgwSsXT9qFmkeavRiZPDD1/CZ34s="
```
Comment (sh1l0n) [inline meetupselector/settings/dev.py:7]: Secret

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+django_settings_module = "meetupselector.settings.dev"
+
+[[tool.mypy.overrides]]
+module = [
+	'celery.*',
+	'debug_toolbar.*',
+	'setuptools.*',
+	'decouple.*',
+]
+ignore_missing_imports = true
```
Comment (sh1l0n) [inline pyproject.toml:32]: False

File: requirements.txt [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Django
```
Comment (sh1l0n) [inline requirements.txt:1]: Why use requeriments file is the project is using poetry? Much better to include dependencies in .toml

**Review by sh1l0n — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 7 above)
Comment (serfer2) [reply requirements.txt:1]: This project is not using poetry at all. pyproject.toml file is a configuration file used, not only for poetry, but also for other tools like `black`. So in this case, requirements file is needed.

File: config/django.dev.env (hunk 2 above)
Comment (serfer2) [reply config/django.dev.env:3]: My suggestion is to create a single `.env` (example.env) file in project root folder, with all environment variables.
That `.env` file is automatically read by Docker when creating containers and is used **only for development**. Developers shall create their own `.env` file in localhost using `example.env` file as a template. 
So, `example.env` is sync with git and `.env` should be added to `.gitignore`.
Then, in CI, when deploying to production, all environment variables are created on the fly.

File: meetupselector/settings/dev.py (hunk 5 above)
Comment (serfer2) [reply meetupselector/settings/dev.py:7]: Suggested change:  
`SECRET_KEY = os.getenv("SECRET_KEY")`

File: Makefile (hunk 1 above)
Comment (aalmiramolla) [reply Makefile:2]: ¡Perfecto, lo cambio!

File: requirements.txt (hunk 7 above)
Comment (aalmiramolla) [reply requirements.txt:1]: As @serfer2 said, the pyproject.toml is not only for poetry, you can use it to configure linters, plugins, etc... I'll close this conversation.

File: pyproject.toml (hunk 6 above)
Comment (aalmiramolla) [reply pyproject.toml:32]: Why? These libraries don't have stubs, so we have to ignore it (https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker)

**Comment by aalmiramolla** [comment]
@sh1l0n, @serfer2 
Can you review the new changes?

**Pull request merged**